brianlong/www.validators.app
# 183010241 fix styling after bootstrap upgrade

#### What's this PR do?
- Fixes styling, js, assets

#### How should this be manually tested?
- Remove cached assets: `rm -r node_modules public/packs public/assets`
- Make sure you use correct version of node (if you're using nvm to manage node versions then you can type in `nvm use` and it should pick up correct version automatically)
- Run `yarn install`
- Click through the app - all pages on desktop and mobile - validators, data centers, stake pools, sol prices, static pages etc., and also user forms (log in, sign up etc.)

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/183010241)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
